dangerfile.js

```diff
@@ -60,6 +90,25 @@ if (componentIntroduced) {
   );
 }
 
+const componentChangedOrCreated = fileChanges.some(filePath =>
+  filePath.match(/packages\/bpk-component.+\/src\/.+\.js/),
```

Comment:
This regex could be extracted, so it's not duplicated (also used a few lines before to get `componentIntroduced`